packages/react/src/badge/Badge.css.ts

```diff
@@ -0,0 +1,92 @@
+import { theme } from "../styles";
+import {
+  type RecipeVariants,
+  createVar,
+  recipe,
+  style,
+} from "../vanilla-extract";
+
+const subtleBackgroundColorVar = createVar();
+const subtleColorVar = createVar();
+const accentBackgroundColorVar = createVar();
+const accentColorVar = createVar();
+
+export const badge = recipe({
+  base: [
+    {
+      alignItems: "center",
+      fontWeight: "500",
+      justifyContent: "center",
```

Comment:
don't need flex related props
```suggestion
      fontWeight: "500",
```

Reply:
Needed the justify content centre Vai for centring the content.